From c4d064b6f02e2f0360133c57ada725c71bca0f2e Mon Sep 17 00:00:00 2001
From: Omair Mohammed Abdullah <omair.m.abdullah@intel.com>
Date: Wed, 7 May 2014 04:18:43 +0530
Subject: [PATCH 33/39] ASoC: Intel: Remove overwriting pcm ID by platform
 driver

Issue: GMINL-3680

PCM ID is not used anywhere else, it is an artifact of legacy code.

Change-Id: Iddf75eb2e0c61d9c406b975fc79956af3870c6f1
Signed-off-by: Omair Mohammed Abdullah <omair.m.abdullah@intel.com>
---
 sound/soc/intel/pcm.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/sound/soc/intel/pcm.c b/sound/soc/intel/pcm.c
index 3fe7f47..7931691 100644
--- a/sound/soc/intel/pcm.c
+++ b/sound/soc/intel/pcm.c
@@ -512,8 +512,6 @@ static int sst_media_prepare(struct snd_pcm_substream *substream,
 	ret_val = sst_platform_alloc_stream(substream, dai->platform);
 	if (ret_val <= 0)
 		return ret_val;
-	snprintf(substream->pcm->id, sizeof(substream->pcm->id),
-			"%d", stream->stream_info.str_id);
 
 	ret_val = sst_platform_init_stream(substream);
 	if (ret_val)
-- 
1.9.1

